lib/merit/models/active_record/merit/sash.rb

```diff
@@ -25,5 +25,9 @@ def score_points(options = {})
       end
       scope
     end
+
+    def badge_count
+      badges_sashes.count
+    end
```

Comment:
I see this avoids "multiple roundtrips to the database". Could you show the logs please? It's maybe a matter of adding an `includes` clause in the existing function, and then we fix a performance issue without changing the API.